guides/common/modules/con_managing-flatpak-repositories-in-project.adoc

```diff
@@ -0,0 +1,12 @@
+[id="managing-flatpak-repositories-in-project"]
+= Managing Flatpak repositories in {Project}
+
+Flatpak enables users to install, manage, and run containerized applications on {Project}. 
```

Comment:
"Containerized" only applies to OCI flatpaks, which Katello supports. There are another flavor of Flatpaks (ostree-style) that we don't support.

The general gist of Flatpak is to manage software, typically for desktop machines instead of headless servers, via portable images.